tools/hidl/hidlgram.y

```diff
@@ -804,12 +803,7 @@ Module
 
 
 ModuleStart
- : MODULE ID
- {
-   CurModule = new ModuleInfo($2.getName());
-   LastProc = NULL;
- }
- | SCMINTERFACE ID
+ : SCMINTERFACE ID
```

Comment:
@afishbeck any backward compatibility concerns removing module and scmclarion?

Reply:
@ghalliday If we are able to build without the *.ecm files that this PR has removed then I don't think those keywords are used anywhere any longer.  It used to build clarion compatible code and HRPC related code for the defined interface.  We used to do quite a few of our binary internal interfaces this way so that we could support those things; all but these few files were converted to be directly writen in c++ a long time ago.